Core/src/org/sleuthkit/autopsy/casemodule/datasourceSummary/DataSourceSummaryPanel.java

```diff
@@ -118,11 +119,11 @@ private String getOSName(DataSource selectedDataSource) {
                     //assumes only one Operating System per datasource
                     //get the datasource id from the OSInfo's first artifact if it has artifacts
                     if (!osInfo.getArtifacts().isEmpty() && osInfo.getArtifacts().get(0).getDataSource().getId() == selectedDataSource.getId()) {
-                        osName = osInfo.getOSName();
-                        //if this OSInfo object has a name use it otherwise keep checking OSInfo objects
                         if (!osName.isEmpty()) {
-                            break;
+                            osName += ", ";
                         }
+                        osName += osInfo.getOSName();
+                        //if this OSInfo object has a name use it otherwise keep checking OSInfo objects
```

Comment:
Maybe remove this old comment and add something up a few lines like "If this OSInfo object has a name add it to the list of OS names for this data source" (assuming that's accurate)

Reply:
I will remove the comment for now, this is adjusted again in the stories I am currently working on so that it no longer uses the OSInfo class or OSUtility, so a revised comment would be extremely short lived.